src/Entity/EntityId.php

```diff
@@ -17,6 +18,30 @@
 
 	protected $serialization;
 
+	const PATTERN = '/^:?(\w+:)*[^:]+\z/';
```

Comment:
Didn't we agree this morning to remove the prefix matching part from this pattern?